### ISSUE TITLE
Push Notification - Near Me

### ISSUE THREAD

**Issue by jberkey78** [entaq/floq#26]
Push notification telling a user there is a CLIQ near by

Interaction - TAP takes user to Near Me section of app

string - "Hey, we found a clip near by."  

ONLY for users with push notifications turned on

**Comment by codesworth** [comment]
I think this will require some server work. Will look into it and see how expensive it may be for our Firebase plan. If necessary we can move it into the next update. 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by jberkey78** [comment]
yeah, it's also not a necessary feature for current build, but will be a nice to have.  I probably should have put it in the doc instead, but worth looking in to.

**Comment by codesworth** [comment]
Alright

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>
